velox/exec/tests/PrestoQueryRunnerTest.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/exec/tests/utils/PrestoQueryRunner.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+#include "velox/parse/TypeResolver.h"
+#include "velox/vector/tests/utils/VectorTestBase.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+
+namespace facebook::velox::exec::test {
+
+class PrestoQueryRunnerTest : public ::testing::Test,
+                              public velox::test::VectorTestBase {
+ protected:
+  void SetUp() override {
+    velox::functions::prestosql::registerAllScalarFunctions();
+    velox::aggregate::prestosql::registerAllAggregateFunctions();
+    velox::parse::registerTypeResolver();
+  }
+};
+
+TEST_F(PrestoQueryRunnerTest, DISABLED_basic) {
```

Comment:
nit: can you add a comment here that this is disabled because it requires a presto co-ordinator running at localhost etc ?

Reply:
Added comments.